demo-apps/cf-plugtest-server/src/main/java/org/eclipse/californium/plugtests/PlugtestServer.java

```diff
@@ -46,92 +60,203 @@
-    		.setInt(NetworkConfig.Keys.NOTIFICATION_CHECK_INTERVAL_TIME, 30000)
-    		.setString(NetworkConfig.Keys.HEALTH_STATUS_PRINT_LEVEL, "INFO");
-        
-        // add resources to the server
-        add(new DefaultTest());
-        add(new LongPath());
-        add(new Query());
-        add(new Separate());
-        add(new Large());
-        add(new LargeUpdate());
-        add(new LargeCreate());
-        add(new LargePost());
-        add(new LargeSeparate());
-        add(new Observe());
-        add(new ObserveNon());
-        add(new ObserveReset());
-        add(new ObserveLarge());
-        add(new ObservePumping());
-        add(new ObservePumping(Type.NON));
-        add(new LocationQuery());
-        add(new MultiFormat());
-        add(new Link1());
-        add(new Link2());
-        add(new Link3());
-        add(new Path());
-        add(new Validate());
-        add(new Create());
-        add(new Shutdown());
-    }
-    
-    
-    // Application entry point /////////////////////////////////////////////////
-    
+	// exit codes for runtime errors
+	public static final int ERR_INIT_FAILED = 1;
+	private static final char[] KEY_STORE_PASSWORD = "endPass".toCharArray();
+	private static final String KEY_STORE_LOCATION = "certs/keyStore.jks";
+	private static final char[] TRUST_STORE_PASSWORD = "rootPass".toCharArray();
+	private static final String TRUST_STORE_LOCATION = "certs/trustStore.jks";
+	private static final String SERVER_NAME = "server";
+	private static final String PSK_IDENTITY_PREFIX = "cali.";
+	private static final byte[] PSK_SECRET = ".fornium".getBytes();
+	private static final int MAX_RESOURCE_SIZE = 8192;
+
+	private static final NetworkConfig CONFIG = NetworkConfig.getStandard();
+	
+	// allows port configuration in Californium.properties
+
+	public static void main(String[] args) {
+		CONFIG // used for plugtest
+		.setInt(NetworkConfig.Keys.MAX_MESSAGE_SIZE, 64).setInt(NetworkConfig.Keys.PREFERRED_BLOCK_SIZE, 64)
+		.setInt(NetworkConfig.Keys.NOTIFICATION_CHECK_INTERVAL_COUNT, 4)
+		.setInt(NetworkConfig.Keys.NOTIFICATION_CHECK_INTERVAL_TIME, 30000)
+		.setInt(NetworkConfig.Keys.HEALTH_STATUS_INTERVAL, 300)
+		.setString(NetworkConfig.Keys.HEALTH_STATUS_PRINT_LEVEL, "INFO");
+
+		if (CONFIG.getInt(NetworkConfig.Keys.MAX_RESOURCE_BODY_SIZE) < MAX_RESOURCE_SIZE) {
+			CONFIG.setInt(NetworkConfig.Keys.MAX_RESOURCE_BODY_SIZE, MAX_RESOURCE_SIZE);
+		}
+
+		
+		// create server
+		try {
+			PlugtestServer server = new PlugtestServer();
+			// ETSI Plugtest environment
+			// server.addEndpoint(new CoAPEndpoint(new InetSocketAddress("::1",
+			// port)));
+			// server.addEndpoint(new CoAPEndpoint(new
+			// InetSocketAddress("127.0.0.1", port)));
+			// server.addEndpoint(new CoAPEndpoint(new
+			// InetSocketAddress("2a01:c911:0:2010::10", port)));
+			// server.addEndpoint(new CoAPEndpoint(new
+			// InetSocketAddress("10.200.1.2", port)));
```

Comment:
Comment formating is a bit strange here.
Should we keep thoses comments ?

Reply:
@mkovatsc 

Do you want to keep it, or could it be removed?